content/blog/almalinux-accounts-launch.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Introducing AlmaLinux Accounts"
+type: blog
+author: 
+    name: "Jonathan Wright"
+    bio: "-"
+    image: /users/jonathan.jpg
+date: '2023-05-03'
+post:
+    title: "Introducing AlmaLinux Accounts"
+---
+Over the past few months, our [infrastructure team](https://wiki.almalinux.org/sigs/Infrastructure.html) has been hard at work preparing our single sign-on system that will provide a unified login experience to nearly all AlmaLinux websites and properties.  We are happy to announce the launch of https://accounts.almalinux.org!
+
+[AlmaLinux Accounts](https://accounts.almalinux.org) will give you a single account to use across the array of sites and services offered by AlmaLinux!  In addition it eases the process of onboarding new contributors through centralized groups and permissions.
+
+## Why?
+Introducing our Single Sign-On (SSO) system offers numerous benefits for both users and the AlmaLinux OS Foundation. SSO enhances security, streamlines access, and simplifies user management across our ecosystem. With a single set of credentials, users can seamlessly navigate all AlmaLinux websites and services. This unified login experience ultimately boosts productivity, fosters collaboration, and provides a more cohesive and personalized experience for our community.
+
+## Software Stack
+In the spirit of FOSS, it was only natural for us to stick with a free and open-source software stack. The chosen components should be familiar to anyone who has dealt with directory management or SSO in the Enterprise Linux space.  We took inspiration from Fedora's account system for some components.
+
+We have chosen the following FOSS components to power our SSO:
+
+- [FreeIPA](https://www.freeipa.org) for directory and user management.
+- [Noggin](https://github.com/fedora-infra/noggin), a web front-end for user self-service. We've been doing work in the Fedora project to get noggin and it's dependencies added to EPEL 9. This will benefit anyone looking to deploy noggin on RHEL 9 or RHEL 9 compatible distros. <sup>1</sup>
```

Comment:
```suggestion
- [Noggin](https://github.com/fedora-infra/noggin), a web front-end for user self-service. We've been doing work in the Fedora project to get noggin and its dependencies added to EPEL 9. This will benefit anyone looking to deploy noggin on RHEL 9 or RHEL 9 compatible distros. <sup>1</sup>
```